api/models.py

```diff
@@ -185,3 +185,29 @@ class SecurityUser(User):
 
     class Meta:
         verbose_name = "Security User"
+
+
+class CheckinCheckout(models.Model):
+    """Stores checkin/Checkout asset logs"""
+    CHECKIN = "Checkin"
+    CHECKOUT = "Checkout"
+
+    option = (
+        (CHECKIN, "Checkin"),
+        (CHECKOUT, "Checkout"),
+    )
+    item = models.ForeignKey(Item,
+                             null=False,
+                             on_delete=models.PROTECT)
+    security_user = models.ForeignKey(SecurityUser,
```

Comment:
can we name this field `logged_by` or `checked_by`?

Reply:
I agree, working on it